diff --git a/mobile/app/build.gradle.kts b/mobile/app/build.gradle.kts
--- a/mobile/app/build.gradle.kts
+++ b/mobile/app/build.gradle.kts
@@ -101,7 +101,7 @@ dependencies {
         exclude(group = "com.google.protobuf", module = "protobuf-javalite")
     }
     implementation(libs.protobuf.java)
-    implementation(libs.tensorflow.lite)
+    implementation(libs.litert)
     implementation(libs.androidx.navigation.compose)
     implementation(libs.androidx.hilt.navigation.compose)
     implementation(libs.androidx.compose.ui.tooling.preview)


diff --git a/mobile/gradle/libs.versions.toml b/mobile/gradle/libs.versions.toml
--- a/mobile/gradle/libs.versions.toml
+++ b/mobile/gradle/libs.versions.toml
@@ -20,7 +20,7 @@ navigationCompose = "2.8.7"
 hiltNavigationCompose = "1.2.0"
 cameraX = "1.4.2"
 mediaPipeTasksVision = "0.10.33"
-tensorflowLite = "2.17.0"
+liteRt = "2.1.4"
 protobuf = "4.26.1"
 
 [libraries]
@@ -58,7 +58,7 @@ androidx-camera-camera2 = { group = "androidx.camera", name = "camera-camera2",
 androidx-camera-lifecycle = { group = "androidx.camera", name = "camera-lifecycle", version.ref = "cameraX" }
 androidx-camera-view = { group = "androidx.camera", name = "camera-view", version.ref = "cameraX" }
 mediapipe-tasks-vision = { group = "com.google.mediapipe", name = "tasks-vision", version.ref = "mediaPipeTasksVision" }
-tensorflow-lite = { group = "org.tensorflow", name = "tensorflow-lite", version.ref = "tensorflowLite" }
+litert = { group = "com.google.ai.edge.litert", name = "litert", version.ref = "liteRt" }
 protobuf-java = { group = "com.google.protobuf", name = "protobuf-java", version.ref = "protobuf" }
 
 [plugins]


